db/migrate/20161101142229_add_region_to_edu_form.rb

```diff
@@ -0,0 +1,5 @@
+class AddRegionToEduForm < ActiveRecord::Migration
+  def change
+    add_column :education_benefits_claims, :regional_processing_office, :string
```

Comment:
can you change this to `null: false`, if the form can't be processed or doesn't match the schema the record won't save anyways so i don't see how we could end up with a null value here. we should also populate this field for all existing records